.github/workflows/aspnetcore-sync.yml

```diff
@@ -0,0 +1,72 @@
+name: AspNetCore->Runtime Code Sync
+on:
+  # Manual run
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  issues: write
+  pull-requests: write
+
+jobs:
+  compare_repos:
+    # Comment out this line to test the scripts in a fork
+    if: github.repository == 'dotnet/runtime'
+    name: Compare the shared code in the AspNetCore and Runtime repos and send a PR to Runtime if they're out of sync.
```

Comment:
Seems like a long name for an action 😆 -- maybe we can shorten it a little bit?

Reply:
```suggestion
    name: Sync Code
```